Modules/Filtering/Denoising/include/itkPatchBasedDenoisingImageFilter.hxx

```diff
@@ -23,7 +23,6 @@
 #include "itkResampleImageFilter.h"
 #include "itkIdentityTransform.h"
 #include "itkLinearInterpolateImageFunction.h"
-#include "itkImageFileWriter.h"
```

Comment:
This `#include "itkImageFileWriter.h"` appears to be introduced with commit 65e90f7017e4c215e9210761c20ea1a21baa03da "ENH: A2D2 Denoising", @krismz, May 30, 2012. But even then, itkPatchBasedDenoisingImageFilter.hxx didn't actually use the header 🤷 Looking at https://github.com/InsightSoftwareConsortium/ITK/blob/65e90f7017e4c215e9210761c20ea1a21baa03da/Modules/Filtering/Denoising/include/itkPatchBasedDenoisingImageFilter.hxx